stablehlo/reference/ProcessGrid.h

```diff
@@ -75,14 +75,14 @@ class RendezvousResult {
   /// Iterates through the (ProcessId, Tensor) map entires and returns a vector
```

Comment:
Not sure it will make much of a difference, since this map is eventually copied into a member variable, and I'd imagine a compiler is smart enough to figure out to avoid two copies during construction, but for the sake of consistency with this part of the codebase can make this change.